3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentRegistry.scala

```diff
@@ -58,6 +57,7 @@ import registry.{
 }
 import registry.apirequest.{ApiRequestEnrichment, ApiRequestEnrichmentConfig}
 import registry.sqlquery.{SqlQueryEnrichment, SqlQueryEnrichmentConfig}
+import registry.pii.PiiPseudonymizerEnrichment
```

Comment:
doesn't scalafmt handle reordering imports alphabetically?

Reply:
I don't know all the settings, but I think it only sorts lists of imports from the same package. (e.g. import blah.{a, b, c} )

Reply:
ah ok then we might look into something that does it for us too